recipes/kibana/Dockerfile

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
I believed this Dockerfile was meant create a Kibana image based on our fork of kibana. So it should starts by `
FROM cogniteev/kibana`, isn't it?